versioned_docs/version-21.04/releases/centreon-core.md

```diff
@@ -652,6 +652,20 @@ New release.
 
 ## Centreon Connector Perl
 
+### 21.04.4
+
+Release date: `null`
+
+#### Improvements
+
+- Fixed an issue that could crash the SSH connector
+
+#### Bug fixes
+
+- UNKNOWN status is displayed instead of WARNING and CRITICAL when using the perl connector
```

Comment:
shouldn't this not be in the release note since it is a regression that appeared with the memory leak fix below?